src/cai-log-client.ads

```diff
@@ -1,29 +1,65 @@
+--
+--  @summary Log client interface
+--  @author  Johannes Kliemann
+--  @date    2019-04-10
+--
+--  Copyright (C) 2019 Componolit GmbH
+--
+--  This file is part of ada-interface, which is distributed under the terms of the
+--  GNU Affero General Public License version 3.
+--
 
 with Cai.Types;
 package Cai.Log.Client with
    SPARK_Mode
 is
 
+   --  Checks if C is initialized
+   --
+   --  @param C  Client session instance
+   --  @return True if C is initialized
    function Initialized (C : Client_Session) return Boolean;
 
+   --  Create new Client_Session
+   --
+   --  @return Uninitialized client session
    function Create return Client_Session with
       Post => not Initialized (Create'Result);
 
+   --  Intialize client
+   --
+   --  @param C               Client session instance
+   --  @param Cap             System capability
+   --  @param Label           Session label
+   --  @param Message_Length  Requested maximal message length, may or may not be provided
```

Comment:
maximal -> maximum

"may or may not be provided" -> remove and explain what happens when the default is used or `0` is supplied.

Reply:
In this case I'm not sure if we even need the `Message_Length` parameter. If it doesn't provide any guarantees about the actual provided message length I see no reason to use it at all since message lengths have to be checked anyway. Also while a larger buffer for the block session might provide performance improvements at runtime I cannot see such benefits here.

Reply:
I'm OK with removing that parameter.